modules/ossm-about-multi-cluster-topology.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is used in the following assemblies:
+
+// * install/ossm-multi-cluster-topology.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-multi-cluster-topology_{context}"]
+= About multi-cluster mesh topology 
+
+In a multi-cluster mesh, you install and manage a single {istio} mesh across multiple {ocp-product-title} clusters, enabling communication and service discovery between the services. Two factors determine the multi-cluster mesh topology: control plane topology and network topology. As a result, there are four possible multi-cluster mesh topologies.
```

Comment:
Should this say that there are two options for each of the two topologies? Without this detail, it's not clear why the two factors would result in "four possible multi-cluster mesh topologies".

Reply:
How about this?

Two factors determine the multi-cluster mesh topology: control plane topology and network topology. There are two options for each topology. Therefore, there are four possible multi-cluster mesh topology configurations.

Reply:
Sounds good.

Reply:
Thanks! 

Done.